cybox/common/vocabs.py

```diff
@@ -6,13 +6,58 @@
 from cybox.common import PatternFieldGroup
 from cybox.utils import normalize_to_xml, denormalize_from_xml
 
-INVALID_XSI_TYPE = "bad_xsi_type"
+
+class VocabField(cybox.TypedField):
```

Comment:
Ah, follow ups! I put it in the `vocabs` package because it was strongly related to other vocabulary concepts and requires importing `VocabString`. The imports in `cybox.__init__` might get kinda weird if we import `VocabString` there, or we'll have to import it in `__set__()`.